to_do_robot.rb

```diff
@@ -1 +1,109 @@
-puts "Hello!"
+require_relative 'to_do_item_class.rb'
+require_relative 'to_do_list_class.rb'
+puts "Welcome to your To-Do list!"
+
+class MakeToDo
+  def initialize
+    puts "Ready to create your to-do list?(Yes or No?)"
+    answer = gets.chomp.downcase
+
+    case answer
+    when 'yes'
+      puts "What would you like to name your ToDo list?"
+      name = gets.chomp.capitalize
+      @list = List.new(name)
+      puts "Your ToDo list #{list.name} was created!"
+
+    when 'no'
+      puts "Okay, see you later!"
+    else
+      puts "I'm sorry, I didn't understand you, please enter 'Yes' or 'No'"
+    end
+  end
+  def view
+    list.view_list
+  end
+
+  def list
+    @list ||= List.new #if @list is not nil then create a new list and assign it to the list var
```

Comment:
This is an awesome pattern to get familiar with, but since your init method is creating a list, you might be able to use an `attr_reader` for this instead.